src/components/SideBar/SideBar.tsx

```diff
@@ -149,8 +148,9 @@ const SideBar = ({
         ref={sidebarRef}
         role="container-sidebar"
         className={composeClasses(
-          'shadow-lg border-t-0 box-border overflow-hidden h-full bg-white fixed transition-all delay-75 duration-200 ease-in z-40',
-          expand ? 'w-72' : 'w-0 lg:w-16'
+          'bg-gray-50 shadow-lg border-t-0 box-border overflow-hidden h-full fixed z-40',
```

Comment:
I think the interaction is not more `zIndex`, can you check it? @AntonioFeregrino